language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/signature/SignatureHelpUtil.java

```diff
@@ -673,6 +674,11 @@ private static boolean withinTextRange(int position, @Nonnull NonTerminalNode no
         if (rawType.kind() == CLASS && ((ClassSymbol) rawType).initMethod().isPresent()) {
             functionSymbols.add(((ClassSymbol) rawType).initMethod().get());
         }
+        if (rawType.typeKind() == TypeDescKind.UNION) {
+            ((UnionTypeSymbol) typeDescriptor).memberTypeDescriptors().stream()
```

Comment:
Here we get any of the members. Could there be more than one object types? If so wouldn't this introduce inconsistent behavior?
Also, shall we break the `getFunctionSymbol` into two? This method includes two cases and I believe it is better for clarity to break it into two, one for the function call expression and the other for method call, remote method call and new expression. WDYT?

Reply:
+1. Sure will do

Reply:
> Here we get any of the members. Could there be more than one object types? If so wouldn't this introduce inconsistent behavior? 

It is unlikely there can be more than one object types.. Should we consider the first member  in that case?

Reply:
If there are multiple object types, we simply can replace `.findFirst().ifPresent((objectMember -> ...)` with `forEach(objectMember -> ....)`

Reply:
> > Here we get any of the members. Could there be more than one object types? If so wouldn't this introduce inconsistent behavior?
> 
> It is unlikely there can be more than one object types.. Should we consider the first member in that case?

Then it's fine. For safety, let's add a comment to avoid unnecessary usage of this method.